spec/order_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'order.rb'
+
+describe Order do
+  subject(:order) { described_class.new({ 'beef-chuck brisket sub' => 4.50, 'pastrami salami sub' => 5.00 }) }
+  let(:menu)      { double(:menu)       }
+
+  it 'initialize with empty hash as current order' do
+    expect(order.current).to eq({})
+  end
+
+  it 'let users check that their total correctly sums up all of the dishes in their order' do
+    expect(order.total).to eq "Your total is: £0.00."
+  end
+
+  it 'adds up price of each selection and stores it in the @total attribute of order' do
+    order.current= {"beef-chuck brisket sub"=>3}
+    expect(order.total).to eq "3 beef-chuck brisket subs at £4.50 each.\nYour total is: £13.50."
```

Comment:
Metrics/LineLength: Line is too long. [96/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)